src/wp-includes/blocks.php

```diff
@@ -432,6 +432,7 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 		'usesContext'     => 'uses_context',
 		'selectors'       => 'selectors',
 		'supports'        => 'supports',
+		'blockHooks'      => 'block_hooks',
```

Comment:
When testing this PR with e.g. the [Like Button block plugin](https://github.com/ockham/like-button/releases/tag/v0.3.1), note that it still uses the `__experimental` prefix (i.e. `__experimentalBlockHooks`) in `block.json`, so you'll have to modify that, or change this line here temporarily to

```suggestion
		'__experimentalBlockHooks'      => 'block_hooks',
```

Reply:
Update, there's now a [new version of the Like Button block plugin](https://github.com/ockham/like-button/releases/download/v0.4.0/like-button.zip) that uses `blockHooks` and can thus be used OOTB to test this PR.